application/classes/Ushahidi/Repository/Form/Attribute.php

```diff
@@ -241,28 +241,41 @@ public function getByForm($form_id)
 	 * that does not directly depend on the rows we are fetching at the time but on the
 	 * list of form ids that match a specific query
 	 */
-	public function getByForms($form_ids) {
+	public function getByForms($form_ids, $include_attributes = []) {
 		$sql = "SELECT DISTINCT form_attributes.*, form_stages.priority as form_stage_priority, form_stages.form_id as form_id " .
```

Comment:
FYI - we should really really be using the query builder for this. Don't refactor it now, but also try not to write new code like this.